### DISCUSSION TITLE
Status code :429

### DISCUSSION THREAD

**Discussion by SsabryDev** [eritislami/evobot#601] [Q&A]
How Do I Fix This? is Theres A Way To Fix?

**Comment by wolf-yuan-6115** [comment]
Your bot got rate limited by YouTube, just wait

**Comment by Takiyo0** [comment]
Ah yes, it happens to me when i requested a lot of request. You can try to restart your bot to fix it. Sometimes restarting once doesn't work, i already do this when i got 429 and it always work!
